app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from app.players.dwarves.dwarf import Dwarf
+from app.players.elves.elf import Elf
+from app.players.elves.elf_ranger import ElfRanger
+from app.players.player import Player
+
+
+def calculate_team_total_rating(players: list[Player]) -> int:
+    return sum([player.get_rating() for player in players])
```

Comment:
You do not need convert this to list
```suggestion
    return sum(player.get_rating() for player in players)
```